### ISSUE TITLE
Stopped working recently

### ISSUE THREAD

**Issue by alanshaw** [postcasio/custom-title#28]
Thanks so much for this module, it's really great, at least it was until recently but now my custom title does not show in the title bar :(

...is anyone else having this problem?

I'm using v1.21.0.

**Comment by csernikmarton** [comment]
The same thing happens for me. Not sure since which version exactly, but I noticed a few days ago that it didn’t work.

**Comment by mettaworldjon** [comment]
Same here... Is there a way we can remove it using "style"?

**Comment by dlio** [comment]
Just found this module and was excited to use it but it does indeed seem to be broken at present.  Thanks in advance to anyone who can take a look at what atom update(s) may have borked it.

**Comment by nlozovan** [comment]
I also wanted to test the package and found out it's not working. Looking into it's code, found out that the selector changing the title is wrong - or at least probably Atom changed it's structure and it's "title" tag is deprecated. 
To make it working, I edited /lib/custom-title.coffee file, below line 89:
`document.title = title`
Added following:
`document.getElementsByClassName("title-bar")[0].innerHTML= '<div class="title">'+title+'</div>'`

In other words, just a stupid selector broke everything. 

I am new to Atom and new to pushing updates in Github - if someone knows how to do it, please update it for the rest of the folks. Cheers!

**Comment by aminya** [comment]
We are interested to maintain this repository. I am the owner of @atom-ide-community. I have sent you an invitation. If you can transfer the repository to @atom-ide-community, we will maintain it. 

@postcasio
